frontend/src/app/testQueue/addToQueue/SearchResults.stories.tsx

```diff
@@ -21,10 +21,13 @@ const patient = {
     deviceTypeName: "MegaTester2000 (PCR)",
   },
 };
+type RouterWithFacilityProps = {
```

Comment:
wondering if we should put this in a `types.d.ts` file to call that instead?

Reply:
I have moved this type to a global setup in react-app-env.d.ts